src/api/crypto/blockchain/BalanceOracle.cpp

```diff
@@ -214,50 +214,37 @@ auto BalanceOracle::Imp::process_registration(Message&& in) noexcept -> void
     const auto haveNym = (2 < body.size());
     auto output = opentxs::blockchain::Balance{};
     const auto& chainFrame = body.at(1);
-    const auto nym = [&] {
-        if (haveNym) {
-
-            return api_.Factory().NymID(body.at(2));
-        } else {
+    const auto nym = api_.Factory().NymID(body.at(2));
```

Comment:
What if body size < 3. I see that nym is not use anywhere in that case but it will be exception here.